templates/csharp/SmartContract1.cs

```diff
@@ -10,8 +12,30 @@
 
 public class SmartContract1 : SmartContract
 {
-    public static void Main()
+    public static object Main(string method, object[] args)
+    {
+        if (method == "PutValue") 
+        {
+            return PutValue(args[0] as byte[]);
+        } 
+        else if (method == "GetValue")
+        {
+            return GetValue();
+        }
+        else 
```

Comment:
I'm wondering if we should include a [contract migrate](https://docs.neo.org/docs/en-us/reference/scapi/fw/dotnet/neo/Contract/Migrate.html) method in the sample. I think it is a best practice for any contract that uses storage (so you can later migrate existing storage to a new version of the contract)

Reply:
Example usage of contract migrate: https://github.com/Moonlight-io/moonlight-ico-template/blob/8a2ddfa86374b93f3fcc3a340c4f1abb4e3f44ee/ICOTemplate/Token/Administration.cs#L126